### PR TITLE
tpcc: fix "sum(O_OL_CNT) = # of rows in the ORDER-LINE table" check

### PR DESCRIPTION
**Pull request by nvb**
The check was trying to scan into a value instead of a pointer, which
was throwing the error
```
check consistency failed: sql: Scan error on column index 0: destination not a pointer
```

It was also comparing sum(o_ol_cnt) with the number of rows in the new_order table for
a district instead of the number of rows in the order_line table for that district.

### CODE REVIEW

**Review by jordanlewis — APPROVED** [review]
Whoops. It's possible that I just committed this without actually finishing it, so you might want to double check that that particular check even works/makes sense.

**Comment by nvb** [comment]
It's failing at the moment, so I hope it's just incomplete and not revealing a larger issue.

**Comment by nvb** [comment]
Oh, I think that's because it's comparing `sum(o_ol_cnt)` with the number of rows in the **new_order** table for a district instead of the number of rows in the **order_line** table for a district! I'll fix that in a second commit.

**Comment by nvb** [comment]
PTAL. With this change, all checks pass!

File: tpcc/check.go [hunk 1]
```diff
@@ -93,16 +91,21 @@ func checkConsistency(db *sql.DB) error {
 
 	// 3.3.2.4
 	// sum(O_OL_CNT) = # of rows in the ORDER-LINE table for each warehouse/district
-	var sumOOLCnt int
+	var sumOOLCnt, countNewOrderLint int
```
Comment (jordanlewis) [inline tpcc/check.go]: why is this called Lint? just curious

**Review by jordanlewis — APPROVED** [review]
LGTM thanks!

File: tpcc/check.go (hunk 1 above)
Comment (nvb) [reply tpcc/check.go]: Because I'm having trouble spelling "line" today 😄

Comment (jordanlewis) [reply tpcc/check.go]: Oh, I get it, haha

**Pull request merged**